pkg/test/rekor_integration/rekor_monitor_integration_test.go

```diff
@@ -0,0 +1,175 @@
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package integration
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"net/http"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"strings"
+	"syscall"
+	"testing"
+	"time"
+)
+
+func TestTamperedCheckpoint(t *testing.T) {
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+	defer cancel()
+
+	buildDir := t.TempDir()
+	binary := filepath.Join(buildDir, "rekor_monitor")
+	buildCmd := exec.Command("go", "build", "-o", binary, "github.com/sigstore/rekor-monitor/cmd/rekor_monitor")
```

Comment:
Use a relative path to the test package to ensure that the test is running on the actual source code